### DISCUSSION TITLE
Push trigger works but not the schedule one

### DISCUSSION THREAD

**Discussion by galderz** [community/community#26985] [Actions]
<p>Hi,</p>
<p>I’ve set up a GitHub action with 2 triggers, push and schedule:</p>
<p><a href="https://github.com/mandrel/graal/blob/mandrel-master/.github/workflows/repo-sync.yml#L8-L9" rel="nofollow noopener">https://github.com/mandrel/graal/blob/mandrel-master/.github/workflows/repo-sync.yml</a></p>
<p>However, only the push one seems to work, not scheduling happens. Any idea what is going on? According to <a href="https://github.community/t/multiple-event-triggers-with-config/16408">https://github.community/t5/GitHub-Actions/Multiple-event-triggers-with-config/td-p/33258</a> multiple triggers are possible.</p>
<p>Thanks</p>
<p>Galder</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/galderz">@galderz</a> ,</p>
<p><a href="https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule" rel="nofollow noopener">Scheduled</a> event only triggers workflow on default or base branch, however your branch(mandrel-master) is not.</p>
<p>As an alternative, you can create a workflow yaml with scheduled event on master(default) branch, and checkout the branch(mandrel-master) for the steps.  Code sample as below:</p>
<pre><code>on:
  schedule:
    - cron: '*/15 * * * *'

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
        with:
          ref: mandrel-master
</code></pre>
<p>Thanks.</p>
